spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java

```diff
@@ -26,6 +28,8 @@
 import java.io.IOException;
 import java.util.Arrays;
 
+import static com.microsoft.azure.spring.cloud.autoconfigure.common.AutoConfigureUtils.getServiceName;
```

Comment:
Duplicate as first added import

Reply:
seems IDE did this for me, let me check.